app/views/shared/_validation_request_form_actions.html.erb

```diff
@@ -0,0 +1,2 @@
+<%= form.govuk_submit "Add" %>
```

Comment:
can you make this a [button group](https://design-system.service.gov.uk/components/button/#grouping-buttons)